internal/constants.go

```diff
@@ -210,7 +198,7 @@ func GetNorduserdLogPath(uid string) string {
 
 	if err != nil {
 		log.Printf("users norduserd logs will be stored in %s: %s", LogPath, err.Error())
-		return filepath.Join(LogPath, Norduserd+"-"+uid+LogFileExtension)
+		return filepath.Join(LogPath, Norduser+"-"+uid+LogFileExtension)
```

Comment:
fmt.Sprintf ?

Reply:
What's the difference?

Reply:
ok, minor, just code consistency

Reply:
No problem, I'll update it. Just thought that there is some functional difference I was not aware of.